examples/platform/silabs/display/lcd.cpp

```diff
@@ -194,19 +283,14 @@ void SilabsLCD::SetQRCode(uint8_t * str, uint32_t size)
     }
 }
 
-void SilabsLCD::ShowQRCode(bool show, bool forceRefresh)
+void SilabsLCD::ShowQRCode(bool show)
```

Comment:
What is the use of the parameter `show`? It is unused and always set to true in all `ShowQRCode` calls

Reply:
good point, will remove it in a follow up PR.